### DISCUSSION TITLE
Conditional formatting

### DISCUSSION THREAD

**Discussion by davefl67** [starship/starship#4201] [Q&A]
What I'd like to do:
- Change username color if sudo time cache is active
- Change hostname color if SSH

Thanks.

**Comment by mrolli** [comment]
Same here, I'm looking for a possibility to change the color of the output of a module if I'm in a remote session.

**Comment by adam-huganir** [comment]
I have the same use case as @mrolli I am trying to solve since all of my remote machines have the same theme (through a dotfiles repo)

**Comment by tkolleh** [comment]
I have a similar use case. Conditional formatting based on OS appearance - light or dark mode. See [discussion post](https://github.com/starship/starship/discussions/4821).
